src/templates/pycontw-2020/contents/en/speaking/cfp.html

```diff
@@ -0,0 +1,98 @@
+
+<p>All speakers are required to buy their own conference tickets（<a href="http://pyfound.blogspot.com/2017/10/psfs-october-board-meeting.html" target="_blank" rel="noopener">Everyone Contributes Policy</a>）</p>
+
+
+<h2>How to Submit Your Proposal</h2>
+
+<p>You need to <a href="{{ signup_url }}">sign up</a>  a new account on tw.pycon.org. With an activated account, you can fill up the speaker profile and create new proposals through the <a href="{{ dashboard_url }}">My PyCon</a> page.</p>
+
+<p>We encourage you to submit the proposal as early as possible. You are welcomed to submit multiple proposals.</p>
+
+
+<h2>Guidelines for Proposal Submission</h2>
+
+<h3>Talks</h3>
+
+<p>We accept a broad range of Python-related proposals, from academic research to commercial projects and case studies, or soft topics such as running a community, making good communication, mental health, etc.  Basically, if you are reading these words, you should go submit your proposal!</p>
+
+<p>We encourage speakers to talk about your own Python package or application, your experience of learning Python or hosting a Python community etc. Talks on advanced topics are highly welcomed as well. For your inspiration, our committee has suggested that they’d love to see talks on following topics: <em>FinTech, Community, A.I, Data Analytics, IoT, Machine Learning, Testing</em>, etc.</p>
+
+<p>When considering different topics, you may be interested in reviewing selected speeches over the past few years at PyCon Taiwan.</p>
+
+<ul>
+  <li><a href="https://tw.pycon.org/2019/en-us/events/schedule/">PyCon Taiwan 2019</a></li>
+  <li><a href="https://tw.pycon.org/2018/en-us/events/schedule/">PyCon Taiwan 2018</a></li>
+  <li><a href="https://tw.pycon.org/2017/en-us/events/schedule/">PyCon Taiwan 2017</a></li>
+  <li><a href="https://tw.pycon.org/2016/en-us/events/schedule/">PyCon Taiwan 2016</a></li>
+</ul>
+
+<p>Talks will be lasting either <em>30 minutes</em> or <em>15 minutes</em>, depends on you, you'll measure and decide how much time you need.  Note that the length of a talk <em>includes setup and Q&amp;A session</em>.</p>
+
+<p>Talks can be given in either <em>English, Mandarin or Taiwanese Hokkien.</em></p>
+
+<p>If it's your first time to propose a talk at PyCon Taiwan or a conference in general, please have a look at <a href="{{ speaking_talk_url }}">How to Propose a Talk?</a> to learn more about conventions, and it might help you organise your thoughts on your proposal.</p>
+
+<p>If your submissions are not selected in PyCon Taiwan, you can choose whether you want to share the talks at the local communities. PyCon Taiwan will provide partial transportation fee. </p>
+
+<h3>Tutorials</h3>
+
+<p>Similar to talks, we don’t pose limitations on tutorial topics.
+  Tutorials are paid, half or full day long, equivalent to 3 or 6 hours of teaching.
```

Comment:
This year, tutorials are free and the length has been changed to 1 hour and 30 mintues.

Reply:
OK, fixed :)